JSDemos/Demos/TreeList/Reordering/description.md

```diff
@@ -1 +1,7 @@
-At&nbsp;first launch, the order of&nbsp;columns in&nbsp;TreeList is&nbsp;the same as&nbsp;the order of&nbsp;objects in&nbsp;the **columns** array. At&nbsp;runtime, a&nbsp;user may be&nbsp;able to&nbsp;reorder columns using drag and drop. To&nbsp;provide the user with this capability, set the **allowColumnReordering** property to _true_.
\ No newline at end of file
+The structure of [dataSource](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#dataSource) objects defines the order of TreeList columns. You can change the order programmatically and allow users to reorder columns in the UI.
```

Comment:
> The structure of [dataSource](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#dataSource) objects defines the order of TreeList columns.

First the old ("the order of columns"), then the new ("is defined by the structure of data objects").

Also, there's little information to divide the description into subheadings, so I moved information from the following paragraph here.

```suggestion
The order of TreeList columns is the same as the order of fields in data objects. You can use the [columns](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/) array to specify a different order. To reorder a column at runtime, change its [visibleIndex](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#visibleIndex) property.
```